Gemfile.lock

```diff
@@ -130,6 +130,7 @@ PLATFORMS
 DEPENDENCIES
   biran!
   bundler (~> 2.1)
+  nokogiri (~> 1.15)
```

Comment:
Could have used 1.14, but I figured why not go to latest since it didn't introduce any other requirements